wear_screenshot_stitch.py

```diff
@@ -74,49 +79,102 @@ def get_row_hashes(image_id):
     row_hashes.append(row_hash)
   return row_hashes
 
+def padded_index(max, num):
+  index_chars = int(math.ceil(math.log10(max)))
```

Comment:
Recommend avoiding float math -- rounding can cause subtle bugs.

Does str(num).zfill(len(str(max)) work instead?

Reply:
No because we're 0-based so len(100) will result in indexes of 000-099 which is silly.